src/Psr/Tracing/RelayOpenTelemetry.php

```diff
@@ -56,14 +56,12 @@ public function __call(string $name, array $arguments)
             ->startSpan();
 
         try {
-            $result = $this->relay->{$name}(...$arguments);
-            $span->end();
-
-            return $result;
+            return $this->relay->{$name}(...$arguments);
         } catch (Exception $exception) {
             $span->recordException($exception)->end();
```

Comment:
@vmihailenco: Since we're calling `end()` here, is the `finally` block actually needed?